frontend/server/src/main/java/com/amazonaws/ml/mms/openapi/OpenApiUtils.java

```diff
@@ -99,10 +98,9 @@ private static Path getPingPath() {
                 "status", new Schema("string", "Overall status of the Model Server."), true);
         MediaType mediaType = new MediaType(HttpHeaderValues.APPLICATION_JSON.toString(), schema);
 
-        Response resp = new Response("200", "Model server status.", mediaType);
-
         Operation operation = new Operation("ping");
-        operation.addResponse(resp);
+        operation.addResponse(new Response("200", "Model server status", mediaType));
```

Comment:
Could we use HttpResponseStatus instead of hardcoded strings for codes?

Reply:
I agree its verbose, but wouldn't that be easier and avoids typo errors? I meant all the strings that we give in this file.

Reply:
Is 500 more readable than String.valueOf(HttpResponseStatus.INTERNAL_SERVER_ERROR.code()) ?

Reply:
500 is more readable, but using standard constants tell what the error (in this case INTERNAL_SERVER_ERROR) is that's being sent out and avoids any error in an accidental change in the future right?

Reply:
We have unit-test to prevent accidental change.